### ISSUE TITLE
[Feature] Add Odometry messages to view plots of pose/speed

### ISSUE THREAD

**Issue by SteveMacenski** [dheera/rosshow#9]
May help in debugging EKFs or generally just odometry messages from encoders in a base. I will keep this in the back of my mind to implement if I have the cycles but wanted to get out in writing if I don't happen to get to it.

**Comment by dheera** [comment]
Yup implementing Odometry, Trajectory, Path, are all on my wish list too. :)

**Comment by SteveMacenski** [comment]
I was thinking of an ascii wheel rotating proportional to speed with all the  data next to it to be fun :)

**Comment by dheera** [comment]
I've added initial support for Odometry messages. It currently only displays 2D Odometry for now (sorry drone users...). I'm thinking about how to best auto-center/auto-scale.

It also displays only the Pose and not the Twist. Your idea of an ascii wheel is quite interesting! I'll definitely think about some form of visualization of the Twist as well. The only issue with a rotating wheel is the frame rate is limited when used over a slow bandwidth SSH connection, so it may not be as easy to discern what the speed is in a useful way. Thoughts?

**Comment by SteveMacenski** [comment]
Oh, I didnt think it would be useful at all, I just figured it'd be cute. Then putting the speed below the wheel as the actual thing of use. This all relative to what's slow or fast but it made more sense to me than a status bar because the range on that is dependent on application. But a wheel spinning people for different applications would get an intuition of how fast it _should_ be for what they're used to without having to place a strict limit or asking for a range
